### ISSUE TITLE
Refactoring: Rename refactoring has problems

### ISSUE THREAD

**Issue by jecisc** [guillep/pharo-core#14830]
Rename refactoring has at least two problems if your new method name already exists in subclasses.

First the dialogue displays a "\n" instead of a line return.

But the most annoying is that if this is the case you can browse the methods with the same name, but you cannot apply the renaming. We could do that before :( 

@balsa-sarenac @Ducasse

**Comment by Ducasse** [comment]
## First point

First the dialogue displays a "\n" instead of a line return. 
- Yes now the question is how to display multiple lines  in Spec dialog because before we could do it and we do not know how to do it now ?

## Second point 
I do not get the second point. Can you elaborate?

**Comment by jecisc** [comment]
For the second point, if you rename method #a into #b and one of the subclass implements #b, then it is impossible to apply a rename. I know that in that case it is not a refactoring but a transformation, but I really wanted to apply this transformation. 

In my case I did a type in the name of my method in the super class and wanted to rename it. But I could not since I already implemented a specialized method in a subclass
